### ISSUE TITLE
TS Lint support

### ISSUE THREAD

**Issue by RyanMillerMath** [revelrylabs/lintron#106]
Are you guys cool with me adding TS Lint support to Lintron? I think it's always cool to expand out the list of supported linters.

**Comment by prehnRA** [comment]
Yes, that would be great. Adding new linters should be relatively straight forward if you emulate the linters that are there already.

**Comment by RyanMillerMath** [comment]
Yeah, I think I have it coded up already. I've been struggling with getting my own version of Linty up & running on Heroku to test it out though. I'll PR it after I figure out how to get my own instance running & have the TSLint functionality verified.
